Jenkinsfile

```diff
@@ -89,9 +93,18 @@ node('linux && docker') {
 
       stage('Deployment pipeline upload') {
         lerna = readJSON file: 'lerna.json'
-        prereleaseVersion = lerna.version
-        version = prereleaseVersion.split('-alpha')[0]
-        sh "deployment-package package create --with-deploy --resolve COMMIT_HASH=${commitHash} --resolve VERSION=${version} --resolve PRERELEASE=${prereleaseVersion}  || true"
+        version = lerna.version
+        releaseVersion = version.split('-alpha')[0]
+        sh """\
+          deployment-package package create
+            --with-deploy
+            --package-name ui-kit/v${version}
+            --target-environment ${isRelease ? 'qa' : 'dev'}
```

Comment:
In a `gitflow` workflow, this `target-environment` parameter allows `dev` to be skipped altogether. This is necessary to start the `dev` pipeline exclusively on alpha bump commits and the `qa`+`prd` pipelines exclusively on release bump commits.

Reply:
I see, so I think the problem is on going into QA, there is a version bump to graduate the release.

However, with two deployment pipeline packages, this means one would approve to go to QA on one pipeline, but then release to prod using a second pipeline? Would that be correct?

If so, I feel it could be confusing, and would prefer if we could do it in one. I think the problem is the graduation version bump, which changes the commit to release?

Would it simplify things if we removed `-alpha` versions all together, and just followed semantic versioning? These versions would initially be tagged on npm as `alpha`. The idea being if a specific version is prod-ready, then we can just change it's npm tag, rather than needing to graduate it's version in a separate commit and start a new pipeline.

Reply:
Essentially yes, approving an alpha bump creates a release bump which goes directly to the qa pipeline and requires QA approval before finally moving on to prod.

If we remove `-alpha`, we can go back to a single deployment package. The only negative then would be that versions read inside the build wouldn't have any indicator of whether they're alpha or not.

Reply:
What exactly is an 'alpha' bump?

Reply:
An alpha bump is a version bump where the version ends with `-alpha`.

Regarding the indicator, I wonder how useful it is because most people will never install these packages. For example, if

1.0.0 is latest

then we publish 1.1.0 with an alpha tag
then we publish 1.2.0 with a latest tag

I expect someone upgrading will go straight to 1.2.0,

Generally, I think all versions are relatively stable, since even those released on merge into master must pass FTs.
With that backdrop, I was wondering if we can make the release process simpler since I think jumping from one pipeline to the next is not dev friendly,